File: 2718/2718.go
func sumOfMatrixAfterQueries(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := sumOfMatrixAfterQueries(root.Left)
	r := sumOfMatrixAfterQueries(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 2718/2718.py
class Solution:
    def sum_of_matrix_after_queries(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.sum_of_matrix_after_queries(root.left), self.sum_of_matrix_after_queries(root.right))